src/components/ReleaseMonitor/__tests__/ReleasesInNamespace.spec.tsx

```
@@ -0,0 +1,265 @@
+            },
+            spec: { target: 'target-ns' },
+        },
+    ];
+
+    const mockReleasePlanAdmissions = [
+        {
+            metadata: { name: 'rpa-1', namespace: 'target-ns' },
+            spec: {
+                data: {
+                    releaseNotes: {
+                        product_name: 'Test Product',
```

ESLint

[error] Identifier 'product_name' is not in camel case.

(camelcase)